### PR TITLE
392 help link

### LINKED ISSUE #392: Giving View HELP Link
## Acceptance Criteria

- [ ] There should be a link on the giving view.

## Context

We need to implement a simple help link so our members can view helpful information if they have trouble giving.

### PR DESCRIPTION
**Pull request by IsaacHardy**
Fixes #392.

## Creator

- [ ] Build relevant tests if any apply
- [x] Ensure there are no new warnings
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer

## Reviewer

- [x] Run `test` and make sure all tests pass
- [x] Review function and form on iOS
- [x] Review function and form on Android
- [x] Review function and form on Web
- [x] Review new test logic

### CODE REVIEW

**Comment by IsaacHardy** [comment]
@burkeshartsis In a future sprint, I'd like to move this link in Heighliner so we can change it without a code update, however, for the sake of time I am implementing a static url.

File: src/give/Now.js [hunk 1]
```diff
@@ -28,15 +28,13 @@ const Now = () => (
...
-            }
+            {/* TODO: Pull this link from Heighliner instead of hard coding */}
             {'Trouble giving? Submit a '}
             <ButtonLink
               onPress={() => WebBrowser.openBrowserAsync('https://rock.newspring.cc/workflows/209')}
             >
-              help request.
-            </ButtonLink>
+              help request
+            </ButtonLink>.
```
Comment (IsaacHardy) [inline src/give/Now.js]: This is new to me. Why is this?

Comment (burkeshartsis) [reply src/give/Now.js]: 1. Because it's ugly.
2. It's bad for SEO. 
3. It's a semi-taught [rule](http://writingspaces.org/wwsg/punctuation-hyperlinks).

Comment (IsaacHardy) [reply src/give/Now.js]: Very nice. TIL

Comment (IsaacHardy) [reply src/give/Now.js]: Oh. I also just noticed what I did. I guess I'm scattered brain today, but I just noticed that I placed the period within the hyperlink. I absolutely knew that was a no no.

**Pull request merged**